unitTestSupportFunctions/calculateDrag.m

```diff
@@ -0,0 +1,30 @@
+%% Function documentation
+%
+% Calculate drag force on a single body with no-slip boundary conditions in a rectangular domain
+%
+%               Input :
+%           FComplete : The complete force vector
+%              fldMsh : Nodes and elements for the fluid mesh
+%              homDBC : The global numbering of the DOFs where homogeneous
+%                       Dirichlet boundary conditions are applied
+%          parameters : Flow parameters
+%
+%
+%              Output :
+%                  Fx : Force (drag) on the body in x-direction
+%
+%% Function main body
+function Fx = calculateDrag (FComplete, fldMsh, homDBC, parameters)
```

Comment:
Add another variable i.e. noDimensions to account for the dimensionality of the problem. For the 2D test cases this variable needs to be set equal to 3 (ux, uy, uz) whereas for 3D cases it should be set to 4.